### ISSUE TITLE
form calculation not showing result in firefox

### ISSUE THREAD

**Issue by LeCube98** [CalderaWP/Caldera-Forms#1871]
Hi
I’m using caldera form as a simple but very efficiant calculator for service purpose.
the form i’ve made works perfectly under chrome but with firefox even if i don’t have any problems going throught all the processed pages it will not show the final result !

it’s some basic values added to each other and a summ at the end and has i can see there is no problem with all the values and fields it’s just not showing the number it should at the end.

Can you help me please ?
Once again it’s only happening under firefox everythings finds under chrome.

regards

Dinosos

Site URL: https://immoservice.nc/calculer

**Comment by Shelob9** [comment]
@LeCube98  THanks for using Caldera Forms. Calculation fields are getting a major overhaul for version 1.5.6. If this issue presists after #1877 is merged, please open a new ticket using the full issue template.

**Comment by Shelob9** [comment]
Close via  #1877
